gatling-maven-plugin/src/main/java/io/gatling/mojo/GatlingMojo.java

```diff
@@ -404,6 +400,57 @@ public boolean accept(File pathname) {
     }
   }
 
+  public ArrayList<String> resolveIncludesAndExcludes(String[] compiledClassFiles) {
+    List<Pattern> includes = new ArrayList<Pattern>();
+    List<Pattern> excludes = new ArrayList<Pattern>();
+
+    MojoUtils.arrayAsListEmptyIfNull(this.includes).forEach(patternString -> {
+      String regexString = patternString.replace(".", "\\.");
+      regexString = regexString.replace("*", "([a-zA-Z\\.]*)");
+      includes.add(Pattern.compile(regexString));
+    });
+
+    MojoUtils.arrayAsListEmptyIfNull(this.excludes).forEach(patternString -> {
+      String regexString = patternString.replace(".", "\\.");
+      regexString = regexString.replace("*", "([a-zA-Z\\.]*)");
+      excludes.add(Pattern.compile(regexString));
+    });
```

Comment:
Hey, supported syntax would be * in trailing position only. I will enhance it to work with the other valid java identifiers.